checker/build.xml

```diff
@@ -383,6 +379,7 @@
                 <include name="**/UnitsTools.java"/>
                 <include name="**/SignednessUtil.java"/>
                 <include name="**/I18nFormatUtil.java"/>
+                <include name="${checker.jdk8orhigher.sources}" if="${jdk8orhigher}"/>
```

Comment:
We still want to be able to build with only JDK 7 installed. It would be a larger refactoring of the build to check for all installed JDKs and build differently depending on that.
If the release is done with JDK 7 & 8 installed, we should use JDK 8 as the default, to ensure Opt is included.